integreat_cms/cms/views/events/event_list_view.py

```diff
@@ -110,10 +110,10 @@ def get(self, request, *args, **kwargs):
         query = None
 
         # Filter events according to given filters, if any
-        filter_data = kwargs.get("filter_data")
-        if filter_data is not None:
+        # TODO How to check for a submitted form??
+        if "filter" in request.GET or "query" in request.GET:
```

Comment:
What would a better check for whether there are form data look like?

Reply:
I think we could do this similar to the page filter form:

https://github.com/digitalfabrik/integreat-cms/blob/ab0a89576ae901f4b30aa8e9c65ff43c44654a80/integreat_cms/cms/views/pages/page_tree_view.py#L91-L99

https://github.com/digitalfabrik/integreat-cms/blob/ab0a89576ae901f4b30aa8e9c65ff43c44654a80/integreat_cms/cms/forms/pages/page_filter_form.py#L45-L53

Reply:
Maybe you could even add a base class for all filter forms with such shared functionality